### PR TITLE
check local files for a cleanup 3 Dec 2020

### PR DESCRIPTION
**Pull request by HosameldinMohamed**
According to a PR in the original repo https://github.com/robotology/robots-configuration/issues/209

We would like to remove some local files in the branch.

This PR is to check this files and to decide if it's we should remove them or commit them.

### CODE REVIEW

**Comment by S-Dafarra** [comment]
I think that the modifications to the ``cer.urdf`` should be removed.

File: iCubGenova04/icub_wbd.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
     <xi:include href="wrappers/inertials/head-inertials_wrapper.xml" />
     <xi:include href="wrappers/inertials/head-inertials_wrapper-deprecated.xml" />
     <xi:include href="hardware/inertials/head-inertial.xml" />
-    <xi:include href="hardware/inertials/waist-inertial.xml" /> 
+    <!--xi:include href="hardware/inertials/waist-inertial.xml" /-->
```
Comment (traversaro) [inline iCubGenova04/icub_wbd.xml:50]: This change is fishy. @prashanthr05 any opinion?

File: iCubGenova04/wrappers/MAIS/left_arm-mais_wrapper_nohands.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        
+        <action phase="startup" level="5" type="attach">
+            <paramlist name="networks">
+                <elem name="leftMais"> left_arm-eb26-j12_15-mais </elem>
+            </paramlist>
+        </action>
+
+        <action phase="shutdown" level="5" type="detach" />
+        </device>
+
```
Comment (traversaro) [inline iCubGenova04/wrappers/MAIS/left_arm-mais_wrapper_nohands.xml:16]: Unless I am wrong, this file and the corresponding file for the right are not used anywhere, so they can be removed?

**Comment by HosameldinMohamed** [comment]
CC  @dic-iit/dynamic-interaction-control

File: iCubGenova04/icub_wbd.xml (hunk 1 above)
Comment (prashanthr05) [reply iCubGenova04/icub_wbd.xml:50]: Sometimes it was required to run the waist-inertial independently from the `yarprobotinterface` to ensure that the device was not stopping abruptly (See https://github.com/robotology/yarp-device-xsensmt/issues/8). Which is why this change exists. We should revert back to the default where this line remains uncommented.

Comment (prashanthr05) [reply iCubGenova04/icub_wbd.xml:50]: Looks like I had forgotten to uncomment and leave it at default. I apologize. I suppose I was the only one in the meantime using the XSens IMU, so it shouldn't have affected any other experiments.

**Comment by HosameldinMohamed** [comment]
This PR is too old and became outdated since the superbuild was freshly installed after that in `icub30`. I forgot to close it.

I close this PR, and I think we can also remove the corresponding branch [cleanup/removeLocalFilesDec20](https://github.com/dic-iit/robots-configuration/tree/cleanup/removeLocalFilesDec20).

**Pull request closed without merging**